code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -331,3 +331,20 @@
 	icon_dead = "penny_dead"
 	icon_rest = "penny_rest"
 	holder_type = /obj/item/holder/cat/penny
+/mob/living/simple_animal/cat/crusher // Goodbye Runtime, make way for the new girl in town.
+	name = "Crusher"
+	desc = "Crusher, nicknamed Dr. Crusher, is a young, smart and cuddly cat. She never lets an opportunity pass to receive some pets or prey on some unsuspecting mice."
```

Comment:
```suggestion
	desc = "A cream coloured, young, and cuddly cat, with a small tag on her collar that says \"Dr. Crusher\". She never lets an opportunity pass to receive some pets or prey on some unsuspecting mice."
```

Reply:
I guess it goes better from the tongue. I didnt see it had a collar, honestly.